src/pal/src/init/pal.cpp

```diff
@@ -680,8 +680,17 @@ PAL_IsDebuggerPresent()
     }
 
     return debugger_present;
+#elif defined(__APPLE__)
```

Comment:
I'm not sure HAVE_SYSCTL is the right way to go.  Not every platform that defines HAVE_SYSCTL will have this mib.  From a quick googling, it appears FreeBSD does, but it looks like OpenBSD expects a few additional members in the mib.

Not sure we can unify this cleanly under that?